script/DeployOFTWrapperFacet.s.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.17;
+
+import { DeployScriptBase } from "./utils/DeployScriptBase.sol";
+import { stdJson } from "forge-std/Script.sol";
+import { OFTWrapperFacet } from "lifi/Facets/OFTWrapperFacet.sol";
+
+contract DeployScript is DeployScriptBase {
+    using stdJson for string;
+
+    constructor() DeployScriptBase("OFTWrapperFacet") {}
+
+    function run()
+        public
+        returns (OFTWrapperFacet deployed, bytes memory constructorArgs)
+    {
+        string memory path = string.concat(
+            vm.projectRoot(),
+            "/config/oftwrapper.json"
+        );
+        string memory json = vm.readFile(path);
+        address oftWrapper = json.readAddress(
+            string.concat(".wrappers.", network)
+        );
+
+        if (oftWrapper == address(32)) {
```

Comment:
why are we comparing here to address(32) and not to address(0) ???

Reply:
@satoshi-lifi 
would be nice if you could still respond to this although the PR has been merged already. I was a bit fast with the merging :)

Reply:
when the element doesn't exist in the json, the value is `address(32)` but not `address(0)`